.github/workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           DOCKER_QUIET_BUILD: 1
         run: |
             DOCKER_BUILD_OPTS="--build-arg UBUNTU_VERSION=${{ matrix.os-version }} --build-arg UBUNTU_NAME=${{ matrix.os-name }}" \
-            make -C utils/docker run DOCKER_DEV_OPTIONAL_CMD='env IS_CI_RUNNING=true \
+            make -C utils/docker build run DOCKER_DEV_OPTIONAL_CMD='env IS_CI_RUNNING=true \
```

Comment:
What's the reason for this?
I think this changes the original build flow rationale in the CI, namely to do the whole build from docker.

Reply:
I added this here to also run the build for `ccenv` but let me integrate this better using the `IS_CI_RUNNING` flag